app/_src/networking/transparent-proxy/configuration-on-kubernetes.md

```diff
@@ -0,0 +1,421 @@
+
+{% warning %}
+It’s best to stick to one method whenever possible. Using more than one can make things more complicated and harder to troubleshoot, as it may not be clear where each setting comes from. If you need to combine methods, check the [**Order of Precedence**](#order-of-precedence) to see what the final configuration will look like based on the priority of each setting.
+{% endwarning %}
+
+## Terminology overview
+
+- **Workload**: In this guide, **workload** refers to an application running in a Kubernetes cluster, usually represented by a [**Pod**](https://kubernetes.io/docs/concepts/workloads/pods/). Kubernetes resources like [**Deployment**](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), [**ReplicaSet**](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/), and [**StatefulSet**](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) are used to define and manage these workloads, resulting in one or more **Pods** where the application actually runs.
+
+- **ConfigMap**: In this guide, **ConfigMap** refers to the Kubernetes [**ConfigMap**](https://kubernetes.io/docs/concepts/configuration/configmap/) resource. It’s used to store configuration data as key-value pairs that can be easily accessed by other resources in the cluster, such as Pods, Deployments, and StatefulSets.
+
+## Method 1: Control plane runtime configuration
+
+The control plane runtime configuration is a section of {{ Kuma }}'s configuration that contains important settings for how the control plane operates, including options for the transparent proxy and other key components.
+
+{% tip %}
+For more details, see the [Control Plane Configuration Reference]({{ docs }}/reference/kuma-cp/) under `runtime.kubernetes`.
+{% endtip %}
+
+Some transparent proxy settings can be adjusted here, and **for certain settings, this is the only place they can be modified**.
+
+Currently, it’s best to use the control plane runtime configuration as the main place to store transparent proxy settings that apply to all workloads. In the future, once the [{{ configuration-in-configmap }}](#method-1-control-plane-runtime-configuration) feature is fully supported, we’ll recommend using ConfigMaps for these settings. However, a few restricted settings that are rarely customized will still need to be in the control plane’s runtime configuration.
+
+Below is a subset of the configuration focused on transparent proxy settings, with default values and corresponding environment variables for each field. For further details, see the [Modifying control plane runtime configuration](#modifying-control-plane-runtime-configuration) section.
+
+```yaml
+runtime:
+  kubernetes:
+    injector:
+      cniEnabled: false # KUMA_RUNTIME_KUBERNETES_INJECTOR_CNI_ENABLED
```

Comment:
can we remove env variable equivalents from here? You won't be using them in this mode anyway, and they are just adding noise in my opinion

Reply:
they are being used, and further is the instruction related to that